build.gradle

```diff
@@ -155,7 +155,7 @@ dependencies {
 
     compileOnly 'org.projectlombok:lombok:1.18.12'
 
-    runtime 'org.hsqldb:hsqldb:2.5.0'
+    runtime 'org.hsqldb:hsqldb:2.4.1'
```

Comment:
This change is not related to the underlying ticket.

It would appear due to the clock change at the weekend the CaseNoteRepositoryTest is now failing with date time related issues.  A ticket will be added to the backlog to investigate and revert back to the more recent version.